.github/workflows/docker.yml

```diff
@@ -160,84 +89,57 @@ jobs:
           name: deb4docker
           path: "${{ github.workspace }}/docker/*.deb"
 
-  setup_tags:
```

Comment:
Same can be done with a single line of config: 
```
tags: |
        type=ref,event=tag
        type=raw,value={{branch}}-{{sha}}
        type=sha
```

Also this introduces errors if the hash is only numbers like '1456234'. However uncommon, this results in helm reformating this integer into its scientific representation like '1.456234+e5' or something similar. Not relevant here since the deployment happens without helm. But relevant for other repos with similar patterns.